### PR TITLE
Remove symlinks for tsconfig.json and .mocharc.json

### LINKED ISSUE #216: Tsc error during first build
During the installation, the yarn build gives an error because the tsconfig of algob and runtime packages seem to be invalid 

image

### PR DESCRIPTION
**Pull request by ratik21**
fixes #216

### CODE REVIEW

File: packages/runtime/tsconfig.json [hunk 1]
```diff
@@ -1 +1,10 @@
...
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "./build"
+  },
+  "include": [
+    "src",
+    "../../@types"
+  ]
+}
```
Comment (amityadav0) [inline packages/runtime/tsconfig.json:10]: Please update the license file as well.

Comment (ratik21) [reply packages/runtime/tsconfig.json:10]: thanks

File: packages/runtime/LICENSE [hunk 2]
```diff
@@ -1 +1,28 @@
-../../LICENSE
\ No newline at end of file
+The ALGOB is licensed under the Apache-2.0 license as defined in the
```
Comment (robert-zaremba) [inline packages/runtime/LICENSE:1]: Please copy the license file from the root directory.

**Review by robert-zaremba — APPROVED** [review]
Preapproving - please change the license file.

File: packages/runtime/LICENSE (hunk 2 above)
Comment (robert-zaremba) [reply packages/runtime/LICENSE:1]: so we should just use the Apache-2.0 license.

Comment (ratik21) [reply packages/runtime/LICENSE:1]: ok

**Pull request merged**